locales-pending/emails-all.ftl

```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Strings in this file are not yet final, and thus should not be localised yet.
+
+email-header-logo-alt = { -brand-mozilla-monitor }
+email-header-button-sign-in = Sign in
+
+email-footer-support-heading = Questions about { -brand-mozilla-monitor }?
+email-footer-support-content = Visit our <support-link>Support Center</support-link> for help
+# TODO: Check with flod - do we need a token for "Mozilla Support"?
+email-footer-reason-subscriber = You’re receiving this automated email as a subscriber of { -brand-mozilla-monitor }. If you received it in error, no action is required. For more information, please visit <support-link>{ -brand-Mozilla } Support</support-link>.
+email-footer-source-hibp = Breach data provided by <hibp-link>{ -brand-HIBP }</hibp-link>
+email-footer-logo-mozilla-alt = { -brand-mozilla }
+
+## Monthly overview email
+
+# Variables:
+#   $month (string) - Translated month this email is about, e.g. "April"
+email-monthly-plus-manual-subject = Your { $month } { -brand-monitor } report
```

Comment:
This isn't safe for localization, you're using a piece of date in a string without accounting for possible declension.

What are the changes of changing this to `Your monthly { -brand-monitor } report ({ $month })`, or even a date (e.g. `April 1, 2024`) instead of the month's name?

Reply:
I'll ask, but we don't have content support anymore (:cry:), so I'm not sure. Perhaps we can add a comment telling localisers that they can change it like that if it doesn't work for their locale?

Reply:
Since the string won't be localised yet, it looks like priority is not high enough to fix it. I've at least added a comment to hopefully avoid accidentally submitting it for localisation at some point anyway: cb62b24dc.